scripts/orchestration/experiment_slack_socket_bridge.py

```
@@ -5,23 +5,122 @@
+        "from repo root.",
+        file=sys.stderr,
+    )
+    raise SystemExit(2) from exc
+
+try:
+    from scripts.orchestration import experiment_slack_bridge_audit as _audit
+    from scripts.orchestration import experiment_slack_bridge_config as _config
+    from scripts.orchestration import experiment_slack_bridge_dispatch as _dispatch
+    from scripts.orchestration import experiment_slack_bridge_rendering as _rendering
+    from scripts.orchestration import experiment_slack_bridge_transport as _transport
+    from scripts.orchestration.experiment_slack_bridge_commands import (
```

Flake8 (7.3.0)

[error] 'scripts.orchestration.experiment_slack_bridge_commands._validate_hypothesis' imported but unused

(F401)